### ISSUE TITLE
Serious Error after Update on pure Linux Systems

### ISSUE THREAD

**Issue by tuxpoldo** [mediathekview/plugin.video.mediathekview#115]
After updating from the official Kodi Plugin Repository, the following problem happened:

```
ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following 
error<-- 
                                            - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS! 
                                           Error Type: <type 'exceptions.ImportError'> 
                                           Error Contents: No module named kodiaddon 
                                           Traceback (most recent call last): 
                                             File "/home/dsc/.kodi/addons/plugin.video.mediathekview/service.py", line 29, in <module> 
                                               from resources.lib.service import MediathekViewService
                                             File "/home/dsc/.kodi/addons/plugin.video.mediathekview/resources/lib/service.py", line 12, in <
module> 
                                               from resources.lib.kodi.kodiaddon import KodiService 
                                           ImportError: No module named kodiaddon 
                                           -->End of Python script error report<--
```

Unfortunately the Pull-Request in the official Kodi Repo was buggy because of a strange behaviour of git running on non case sensitive filesystems: the changes from #89 cased many files to be renamed to lower case. When creating the pull request for the official Kodi Repo, git did not transfer the file rename of the files in the directories `resources/lib/base` and `resources/lib/kodi`.

As a workaround the problem can be fixed by renaming all files in both directories to lowercase. I'm working on a new pull request fixing the problem. As an alternative you may also reinstall the version 0.6.0 from the [mediathekview beta repository](https://mediathekview.de/download/) or directly by downloading the [0.6.0 release zip here](https://github.com/mediathekview/plugin.video.mediathekview/releases/download/release-0.6.0/plugin.video.mediathekview-0.6.0.zip).

**Comment by tuxpoldo** [comment]
Pull Request [#2329](https://github.com/xbmc/repo-plugins/pull/2329) in [xbmc/repo-plugins](https://github.com/xbmc/repo-plugins) has been filed.

**Comment by MarcusRoeckrath** [comment]
Thanks, new release is working fine.

**Comment by tuxpoldo** [comment]
Incredible: The Reviewers Team has already merged the pull request [#2329](https://github.com/xbmc/repo-plugins/pull/2329) in [xbmc/repo-plugins](https://github.com/xbmc/repo-plugins)
